Documentation/markdown/SetupHelp.md

```diff
@@ -7,11 +7,13 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 
 1. **Visual Studio 2017** is installed to your computer - find [here](https://my.visualstudio.com/Downloads?q=visual%20studio%202017&wt.mc_id=o~msft~vscom~older-downloads)
 2. **CMake 3.17.0** is installed - find [here](https://cmake.org/download/)
-3. **Qt 5.12.1** is installed and included in path -  find [here](https://www.qt.io/download-open-source)
-	-binaries x64 msvc2017 
-	-mingw
-4. **Boost 1_65_1** is installed and included in path -  find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download)
-5. **Anaconda 2** 
+3. **Qt 5.12.1** is installed and included in the path -  find [here](https://www.qt.io/download-open-source)
+
+	*binaries x64 msvc2017 
+	*mingw
```

Comment:
Denote that this is *mandatory*. The rest is optional and does not influence work with OpenInfraPlatform.

Reply:
Done